generators/Program.fs

```diff
@@ -49,9 +49,11 @@ let private regenerateTestClasses options =
     
     createExercises options
     |> List.filter (shouldBeIncluded options)
-    |> List.iter regenerateTestClass'
-
-    Log.Information("Re-generated test classes.")
+    |> function
+        | [] -> Log.Error "No exercises matched given options."
```

Comment:
Maybe `Log.Warning` would be a better error level on second thought. So sorry about this.